README.md

```diff
@@ -6,86 +6,158 @@ Command Line Interface for Clever Cloud.
 
 ## Installation
 
-⚠️ *We no longer support the usage and installation of Clever Tools via a global npm install.*
+The clever-tools CLI can be installed through many different channels depending on your system setup.
 
-Please follow the installation instructions for your platform (see below) and make sure to uninstall the npm version with this:
+### Via npm
 
-    npm uninstall -g clever-tools
+If you already have node/npm on your system, you can run:
 
-Please refer to <https://www.clever-cloud.com/doc/clever-tools/getting_started/> for the installation details.
+```sh
+npm install -g clever-tools
+```
+
+WARNING: ⚠ This only works for 1.0.0 and after but it is not released yet.
+
+If you want to install our latest beta release, you can run:
+
+```sh
+npm install -g clever-tools@beta
+```
+
+### On GNU/Linux
+
+#### Debian/Ubuntu (.deb)
+
+If you are using a GNU/Linux distribution that uses `.deb` packages like Debian or Ubuntu, you can run:
+
+```sh
+apt-get install apt-transport-https gnupg
```

Comment:
This is most likely installed by default

Reply:
I had to do it on the default docker/debian so I was not sure.
I removed it.